packages/cli/src/cmds/standalone/handler.ts

```diff
@@ -21,7 +21,14 @@ import { IBundlerArgs } from "./index";
 export async function bundlerHandler(
   args: IBundlerArgs & IGlobalArgs
 ): Promise<void> {
-  const { dataDir, configFile, testingMode, unsafeMode, redirectRpc } = args;
+  const {
+    dataDir,
+    configFile,
+    testingMode,
+    unsafeMode,
+    redirectRpc,
+    manualBundling,

```

Comment:
Can't we use bundlingMode itself as the param instead of introducing a new variable?

Reply:
done